### ISSUE TITLE
Fix S2115 FN: Support multi-line string interpolation inside a raw string literal

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6293]
### Description

FN for S2115 (DatabasePasswordsShouldBeSecure).

Using C#11 multi-line string interpolation inside a raw string literal, the rule is not raising an issue.

### Repro steps

```csharp
string[] test = new string[] { "FirstTest", "SecondTest" };
optionsBuilder.UseSqlServer($$"""Server={{test
    [0]}};Database=myDataBase;User Id=myUsername;Password="""); // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
